app/templates/layouts/_error.html

```diff
@@ -14,6 +14,11 @@ <h1 class="saturn">Error {{status_code}}</h1>
         {%- endfor -%}
       </ul>
     {%- endif -%}
+
+      <div class="info__tel">
+          <a class="info__link" href="tel:03001234931" aria-describedby="survey-help">0300 1234 931</a>
```

Comment:
Agreed content on the Trello card is:

`If the problem continues please call 0300 1234 931`

Reply:
OK, will amend